cinema/serializers.py

```
@@ -128,7 +129,7 @@ def validate(self, attrs):
             attrs["row"],
             attrs["seat"],
             attrs["movie_session"].cinema_hall,
-            ValidationError
+            ValidationError,
```

In `TicketSerializer.validate` you call `Ticket.validate_ticket(..., ValidationError)`. Please confirm `Ticket.validate_ticket` expects an exception *class* as its last argument and will raise it as needed. If it expects the validation function to raise directly, consider calling `Ticket.validate_ticket(...)` and then raising `ValidationError(...)` here, or change the helper to raise the provided exception class. This avoids surprising runtime errors or silent failures.